### PR TITLE
Integration to pokedex tool + nicer UI for webview

### PR DESCRIPTION
**Pull request by SwissKid**

### CODE REVIEW

File: decode.py [hunk 1]
```diff
@@ -39,8 +41,21 @@
 request_api = {} #Match responses to their requests
 pokeLocation = {}
 request_location = {}
+def sendToMap(t,uid,point, meta):
+  if bearer == "":
+    return
+  headers = {"Authorization" : "Bearer " + bearer}
+  data = {"type" : t,
+          "uid" : uid,
+          "location" : point.coordinates,
```
Comment (barzamin) [inline decode.py]: @SwissKid you should be passing the whole `point` object not just `point.coordinates`.

**Comment by barzamin** [comment]
Might want to add a flag or config var to turn this on and off.

**Comment by SwissKid** [comment]
There is - if secrets.py doesn't exist, or the bearer token isn't in it, it does a "return"

For the "type: Point", i get an error if I send it in that way (it seemed to only look for the coords)

**Comment by barzamin** [comment]
@SwissKid `type: Point` should work now, the validation schema was wrong before and was rejecting valid geojson.

File: decode.py [hunk 2]
```diff
@@ -41,8 +47,33 @@
...
+  print r.content
+def createItem(t,uid,point, meta):
+  data = {"type" : t,
+          "uid" : uid,
+          "location" : point,
+          "meta" : meta
+  }
+  return data
+  
+def sendToMap(t,uid,point, meta):
```
Comment (bettse) [inline decode.py]: I'm going to drop this when its merged since its not used

File: ui/index.html [hunk 3]
```diff
@@ -37,8 +37,14 @@
 <div id='map'></div>
 <script src="leaflet.js"></script>
 <script src='leaflet-providers.js'></script>
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
 <script src='leaflet-realtime.min.js'></script>
 <script src="index.js"></script>
+<script>
```
Comment (bettse) [inline ui/index.html]: I think I fixed this already when I refactored the filter UI

**Pull request merged**